### DISCUSSION TITLE
% character always getting encoded

### DISCUSSION THREAD

**Discussion by PolEspurnes** [projectdiscovery/fuzzing-templates#10] [Q&A]
Hello,
I tried using the "crlf-injection" template from the repository and I see that in all the fuzzing requests, the character '%' is getting encoded to %25.
For example, payload '%00' is sent as '%2500'
I have also seen this behavior on some fuzzing templates I tried to create by myself and I can't find how would you avoid this encoding.
I would like to send the payload with the % character not getting automatically encoded.
Am I missing something?
The command that I am using:
`nuclei -u<url> -t fuzz-crlf.yaml -p <my-proxy>`

Thanks for your help.

**Comment by ehsandeep** [comment]
@PolEspurnes this will be fixed soon and getting tacked at https://github.com/projectdiscovery/nuclei/issues/2698
